project-base/.eslintignore

```diff
@@ -1,7 +1,3 @@
-/src/Resources/scripts/frontend/plugins
-
-/src/Resources/scripts/styleguide

```

Comment:
Styleguide related files are now also part of a node module?

Reply:
Yes. And I think that we dont have to ignore coding standards of styleguide. 

```js
// package.json
"codemirror": "^5.49.2",
```

```js
// webpack.config.js
.addEntry('styleguide', './assets/js/styleguide/styleguide.js')
```